frontend/src/app/supportAdmin/SupportAdminRoutes.tsx

```diff
@@ -39,6 +40,10 @@ const SupportAdminRoutes: React.FC<Props> = ({ isAdmin }) => {
         path="tenant-data-access"
         element={<TenantDataAccessFormContainer />}
       />
+      <Route path={"unarchive-patient"}>
+        <Route path=":pageNumber" element={<UnarchivePatientContainer />} />
+        <Route path="" element={<UnarchivePatientContainer />} />
```

Comment:
Is there a way to set the route just once and fallback to a default value if no page number is passed?. Also how can the user pass a page number but the facility selected is not needed?.

Reply:
Ahh yes! It looks like there is a way to set the route once!

I think the only way a user can pass a page number without having selected a facility is by manipulating the url and appending a page number but nothing would display since they have not selected an org or facility